distribution/docs/src/main/resources/content/_installing/java-reqs.adoc

```diff
@@ -0,0 +1,96 @@
+:title: Java Requirements
```

Comment:
❓ Was this file meant to be added in this PR?

Reply:
I think it was meant to be in this one: https://github.com/codice/ddf/pull/4043. I'll reconcile them before I merge this one

Reply:
fixed